product/views/ManageIQ_Providers_CloudManager_Template.yaml

```diff
@@ -24,6 +24,7 @@ cols:
 - allocated_disk_storage
 - last_scan_on
 - region_description
+- image?

```

Comment:
I can. I'm not sure why this file (and several other of these yamls) seems to have windows style line endings.